all/experiments/writer.py

```diff
@@ -5,11 +5,18 @@
 from datetime import datetime
 from tensorboardX import SummaryWriter
 from all.logging import Writer
+from comet_ml import Experiment
```

Comment:
This import fails if comet isn't installed. That appears to by why the tests are failing.